recipes/qt5/bld.bat

```diff
@@ -1,104 +1,137 @@
 @echo on
-set ENABLEDELAYEDEXPANSION
+setlocal EnableDelayedExpansion
 set QMAKESPEC=win32-msvc%VS_YEAR%
 set SHORT_VERSION=%PKG_VERSION:~0,-2%
 
-:: NOTE: you must have the DirectX SDK installed for ANGLE to compile:
-::    https://www.microsoft.com/en-us/download/details.aspx?id=6812
-::    (572 MB, not availabe through Conda)
-::   You should also set the DXSDK_DIR environment variable once you have the SDK installed.
-
-IF "%DXSDK_DIR%" == "" (
-    echo You do not appear to have the DirectX SDK installed.  Please get it from
-    echo    https://www.microsoft.com/en-us/download/details.aspx?id=6812
-    echo and try this build again.  If you have installed it, and are still seeing
-    echo this message, please open a new console to refresh your environment variables.
-    exit 1
+:: conda-forge/obviousci's obvci_appveyor_python_build_env.cmd used
+:: so we shouldn't include it finally.  I use it to set the correct
+:: Windows SDK, but conda-build should take care of that probably??
+:: It requires TARGET_ARCH to be one of x64 or x86.  On conda-forge
+:: this script is run *before* bld.bat.
+if "%ARCH%" == "64" (
+  set TARGET_ARCH=x64
+) else (
+  set TARGET_ARCH=x86
+)
+call %RECIPE_DIR%\obvci_appveyor_python_build_env.cmd
```

Comment:
I am not sure, we could talk about integrating `obvci` into `conda-build` somehow, possibly? @pelson and @msarahan, I would welcome your input.